### PR TITLE
Create Thana.md

### PR DESCRIPTION
**Pull request by ThanaMeri**

### CODE REVIEW

File: docs/Thana.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+ 
+![image005](https://user-images.githubusercontent.com/31999574/199713480-00146e0a-5bab-4402-a20c-21287d1a9701.png)
+
+1. يدعم الموقع العديد من اللغات، قمت حسب تجربتي بالترجمة من الإنجليزية إلى العربية.
+
+المهمة الموكلة إلينا هي أن يقوم **كل شخص باختيار ملف ليقوم بترجمته،وهذا الملف هو ترجمة لفيديو معين قمت باختيار ملف Open-Data.srt إلى العربية وقام باقي زملائي باختيار ملفات أخرى**.
+
+# عند فتح الملف المراد ترجمته، تنقسم النافذة إلى ثلاثة أقسام رئيسية:
+![image007](https://user-images.githubusercontent.com/31999574/199715144-83da7733-b8fc-46d6-8d12-22d84e923a90.png)
+* [ ] الجزء الأوسط (2) وهو الجزء الرئيسي، حيث نختار أفضل ترجمة من الأسفل (وهي مترجمة ترجمة آلية)، ونقوم بالتعديل عليها أو إعادة ترجمتها كليًا حسب ما يقتضيه النص بالإضافة إلى الاهتمام بأدوات الترقيم ثم اختيار حفظ ليتم إضافة هذه الترجمة واعتمادها (كما في الصورة المرفقة).
```
Comment (BatoolMM) [inline docs/Thana.md]: This paragraph is aligned left-to right, it should be right-to-left, can you fix it, please?

**Review by BatoolMM — CHANGES_REQUESTED** [review]

**Comment by BatoolMM** [comment]
@all-contributors please add @ThanaMeri for documentation and Translation

File: docs/Thana.md (hunk 1 above)
Comment (ThanaMeri) [reply docs/Thana.md]: I think it's good now

**Comment by BatoolMM** [comment]
Thank you so much @ThanaMeri, there's still a problem with alignment with the 2nd and 3rd points but I will merge it.

**Pull request merged**